samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -146,33 +151,43 @@ static bool identifying_info_read_allow(struct bt_conn *conn)
 		return true;
 	}
 
-	LOG_INF("Rejecting read operation of identifying information");
+	for (size_t i = 0; i < ARRAY_SIZE(uuid_block_list); i++) {
+		if (bt_uuid_cmp(uuid, uuid_block_list[i]) == 0) {
+			uuid_match = true;
+			break;
+		}
+	}
+
+	if (!uuid_match) {
+		return true;
+	}
+
+	LOG_INF("Rejecting operation on the identifying information");
 
 	return false;
 }
 
-static bool gatt_read_authorize(struct bt_conn *conn, const struct bt_gatt_attr *attr)
+static bool gatt_authorize(struct bt_conn *conn, const struct bt_gatt_attr *attr)
 {
-	const struct bt_uuid *uuid_block_list[] = {
-		/* GAP service characteristics */
-		BT_UUID_GAP_DEVICE_NAME,
-	};
+	bool authorized = true;
 
-	for (size_t i = 0; i < ARRAY_SIZE(uuid_block_list); i++) {
-		if (bt_uuid_cmp(attr->uuid, uuid_block_list[i]) == 0) {
-			/* Fast Pair Implementation Guidelines for the locator tag use case:
-			 * The Provider shouldn't expose any identifying information
-			 * in an unauthenticated manner (e.g. names or identifiers).
-			 */
-			return identifying_info_read_allow(conn);
-		}
+	/* Fast Pair Implementation Guidelines for the locator tag use case:
+	 * The Provider shouldn't expose any identifying information
+	 * in an unauthenticated manner (e.g. names or identifiers).
+	 */
+
+	if (IS_ENABLED(CONFIG_APP_DFU)) {
+		authorized = authorized && app_dfu_bt_gatt_operation_allow(attr->uuid);
 	}
 
-	return true;
+	authorized = authorized && identifying_info_allow(attr->uuid);
```

Comment:
Still, you actually use those functions to restrict access to GATT characteristics (if any of the calls block access, you do not allow accessing given GATT characteristic - a given call restricts access to its charactersitcs). If you prefer to keep `_allow` suffix, a function could e.g. early return if access is allowed by any of the call (allow list instead of deny list).

Reply:
discussed, stays as is